lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1362,6 +1362,12 @@ enum Severity {
   eSeverityInfo, // Equivalent to Remark used in clang.
 };
 
+enum NotificationType {
+  eDebuggerWillBeCreated = (1 << 0),
```

Comment:
Should this be `eDebuggerWasCreated` to indicate that a debugger object was created and that the user can do something with it?

Reply:
Yes, it's not useful to call this before the new Debugger is fully initialized.